src/components/LoginPage/LoginPage.jsx

```diff
@@ -1,18 +1,92 @@
-import React from 'react';
-import { Button } from 'react-bootstrap';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
You can go ahead and run `npm i bootstrap` and `npm i react-bootstrap` so you don't have to refer to it like that. We will most likely be using it in the future anyways.